### DISCUSSION TITLE
Synology NAS Indicates "Docker container nginx-proxy-manager stopped unexpectedly" on Every Watchtower Upgrade

### DISCUSSION THREAD

**Discussion by barndawgie** [NginxProxyManager/nginx-proxy-manager#2759] [Q&A]
I'm running Docker on my Synology NAS for a bunch of containers. I recently installed Nginx Proxy Manager and unlike all of my other containers, it is telling me "Docker container nginx-proxy-manager stopped unexpectedly" on every upgrade via Watchtower. Watchtower sends a SIGTERM and gives 60s for a container to shut down. There is nothing obvious in the logs - it appears to shut down normally.

Is there something specific about Nginx Proxy Manager shutdown behavior that I need to be aware of/tweak?

**Comment by MarkKla** [comment]
have you set the restart policy to unless stopped or always?

**Reply by barndawgie** [reply]
> have you set the restart policy to unless stopped or always?

I used `--restart unless-stopped` on my docker run command.

FWIW, I get the same behavior (i.e. notification of unexpected container stop) if I run `docker kill -s SIGTERM nginx-proxy-manager`

If it matters, I did enable the HealthCheck per https://nginxproxymanager.com/advanced-config/#docker-healthcheck
